src/spring-cloud/setup.py

```diff
@@ -16,7 +16,7 @@
 
 # TODO: Confirm this is the right version number you want and it matches your
 # HISTORY.rst entry.
-VERSION = '2.11.1'
+VERSION = '2.11.3'
```

Comment:
Why is the latest version of `HISTORY.md` `2.11.2`, but the version here is `2.11.3`?

Reply:
If the version of `2.11.2` has not been released before, it is recommended not to skip the release of version `2.11.2`

Reply:
That's because we missed `2.11.2` in `HISTORY.md` in the last commit.

Reply:
> If the version of `2.11.2` has not been released before, it is recommended not to skip the release of version `2.11.2`. Maybe we can consider merging the history of two `2.11.1` version, and release this PR as version `2.11.2`

Make sense, updated to `2.11.2` now. Please help to check.